lib/dndsections.sty

```diff
@@ -13,6 +18,14 @@
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 % Section formatting
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+% Part
+\titleformat {\part}
+  [display]
+  { \centering \DndFontPart } % format
+  { \contourlength{0.02em} \contour{lightgray}{\partname\ \thepart} } % label
+  {2ex} % sep
+  {\contour{lightgray}} % before-code
```

Comment:
A `partcontour` color needs to be added to dndcolors.sty and used instead of `lightgray`. A user should be able to change the color without redefining a built-in color or redefining `part`.